src/System.Runtime/tests/System/ActivatorTest.netstandard1.7.cs

```diff
@@ -0,0 +1,101 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.Globalization;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace System.Tests
+{
+
+    public class ActivatorTestsExtended
+    {
+        class ClassWithPrivateCtor
+        {
+            static ClassWithPrivateCtor() { Flag.Reset(100);}
+            private ClassWithPrivateCtor() { Flag.Increase(200);}
+            public ClassWithPrivateCtor(int i) { Flag.Increase(300);}
+        }
+        class ClassWithPrivateCtor2
+        {
+            static ClassWithPrivateCtor2() { Flag.Reset(100);}
+            private ClassWithPrivateCtor2() { Flag.Increase(200);}
+            public ClassWithPrivateCtor2(int i) { Flag.Increase(300);}
+        }
+        class ClassWithPrivateCtor3
+        {
+            static ClassWithPrivateCtor3() { Flag.Reset(100);}
+            private ClassWithPrivateCtor3() { Flag.Increase(200);}
+            public ClassWithPrivateCtor3(int i) { Flag.Increase(300);}
+        }
+
+        [ComVisible(false)]
```

Comment:
It might have some effect in .NET Core, but I do not think we should be testing it in corefx. It should be tested as part of interop tests.

cc @yizhang82